book/content.nim

```diff
@@ -1,23 +1,82 @@
+import std / strformat
 import nimib, nimibook
+import nimib / themes
+from nimibook / commands import emptySrcFile
 nbInit(theme = useNimibook)
 
 nbText: """
 # Content
 
-The content of a chapter of a book can be a standard Markdown file or a nim file.
+The content of a chapter of a book can be a standard Markdown file (`.md`) or a `.nim` file.
 
-For the nim file the structure is the following:
+The nim file must be a nimib document and the default content (created by `init` command) is:
+"""
+nbCode:
+  discard
+nb.blk.code = emptySrcFile("Default Content", ".nim")
 
-```nim
-import nimib, nimibook
-... # other imports
 
-nbInit # initializes a nimib document
-nbUseNimibook # overrides nimib defaults with nimibooks and loads book configuration
+nbText: """## Latex
+
+Latex is available through [Katex](https://www.latex-project.org) as in Nimib.
+It is enabled on a specific document with the command
+"""
+nbCode:
+  nb.useLatex
+
+nbText: &"""
+which adds the following content in the head of the document:
+
+```html
+{themes.latex}
+```
+"""
+
+let inline_paragraph = """
+> Euler's identity is the equality
+> $e^{i\pi} + 1 = 0$, where $e$ is the base of natural logarithms,
+> $i$ is the imaginary unit, and $\pi$ is the ratio of circumference of a circle
+> to its diameter.
+"""
+let block_equation = """
+$$ 
+  f(x) = \frac{1}{\sigma\sqrt{2\pi}} 
+  \exp\left( -\frac{1}{2}\left(\frac{x-\mu}{\sigma}\right)^{\!2}\,\right)
+$$
+"""
+
+nbText: &"""## Inline equations
+
+**Inline equations** are obtained with delimiters `$..$`. The following inline paragraph:
```

Comment:
This is a bit picky, but I read this as you should write equations like `$. e{\pi i} .$`. Adding a third dot and adding spaces makes it more obvious that the points just placeholders.
```suggestion
**Inline equations** are obtained with delimiters `$ ... $`. The following inline paragraph:
```